src/components/sortable/sortable.stories.tsx

```diff
@@ -0,0 +1,266 @@
+import { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import { SortableItem } from './sortableItem';
+import { SortableList } from './sortableList';
+import { DragLayer } from './dragLayer';
+import type { SortableItemProps, DragItem } from '@common/types';
+
+const meta: Meta<typeof SortableItem> = {
+  title: 'DnD/Sortable',
+  component: SortableItem,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  decorators: [
+    (Story) => (
+      <DndProvider backend={HTML5Backend}>
+        <Story />
+      </DndProvider>
+    ),
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj<SortableItemProps>;
+
+interface ListItem {
+  id: number;
+  name: string;
+}
+
+const itemStyle: React.CSSProperties = {
```

Comment:
Please check other similar ones
```suggestion
const itemStyle: CSSProperties = {
```